src/pages/index.js

```diff
@@ -231,7 +230,6 @@ function handleAvatarSubmit(evt) {
   function makeRequest() {
     return api.editAvatarInfo({ avatar: avatarInput.value }).then((data) => {
       avatarImage.src = data.avatar;
-      evt.target.reset();
       closeModal(avatarModal);
```

Comment:
`closeModal(avatarModal);` is not needed here because it's already inside `handleSubmit`